src/styles/constants.scss

```diff
@@ -2,10 +2,62 @@ $primary-font:"Roboto";
 
 $rhythm: 12px;
 
-@mixin flex-center {
-  display: flex;
-  align-items: center;
-  justify-content: center;
+@mixin flex($display: flex,
+  $direction: null,
+  $placeContent: null,
+  $placeItems: null,
+  $wrap: null,
+  $shrink: null,
+  $grow: null,
+  $alignContent: null,
+  $justifyContent: null,
+  $alignItems: null,
+  $justifyItems: null) {
+  display: $display;
```

Comment:
Do you think we should add all the properties here? My idea was to add a bare minimum of them that are mostly used like justify, align, and direction. If you believe you need all of them let's go with them otherwise you can reduce the required code here.

Reply:
Coool, got it.